### ISSUE TITLE
botting blocking quests

### ISSUE THREAD

**Issue by hothedhunter** [dalaranwow/dalaran-wow#5548]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. howling fjord spirits of the ice quest
2. returned to area many times attempting quest, unable to complete on horde toons
3. all adds gone, being killed by botting alliance palidens

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
while attempting to complete this quest on horde, there are 2 alliance palidens with the toon names cemtus and makrata that are making the exact same round every 15 minutes killing all the adds that drop the needed quest item. after waiting for over 24 hours attempting to complete the quest we returned and began killing the palidens, these characters did not fight us back they remained focused on the adds. both toons are around level 72. after killing the toons, they would respawn in the exact spot they were killed within seconds. We have completed the quest, however I do not want this to continue and cause issues for other players.

**Comment by bulletfreak** [comment]
Can verify this, when I ran into them it was two different human paladin bots named "Damoren" and  "Webola" which me and my friends farmed for a while because screw botters. After a while an 80 Human Warrior named "Wilders" killed us and proceeded to chase us away and then /follow the bots for a while until he was sure we were gone. We got on an alliance alt and added him to our friends list to see where he was and if he was still on, he seemed to AFK at Utgarde Pinnacle so he was close to the bots so he could quickly fly there to defend them. The bots occasionally made a bee line to Fort Wildervar to presumably sell whatever it is they're farming.
